### PR TITLE
dev-libs/jansson: version bump to 2.11

### PR DESCRIPTION
**Pull request by Zer0-One**
The 2.11 ebuild removes multilib_src_configure(); the modification to
src/Makefile.am hasn't been relevant for about 3 years now.
Consequently, the autotools include has been removed as well.

### CODE REVIEW

**Comment by Zer0-One** [comment]
@doughdemon done.

File: dev-libs/jansson/jansson-2.11.ebuild [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+inherit multilib-minimal
+
+DESCRIPTION="C library for encoding, decoding and manipulating JSON data"
+HOMEPAGE="http://www.digip.org/jansson/"
+SRC_URI="http://www.digip.org/jansson/releases/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86"
```
Comment (mgorny) [inline dev-libs/jansson/jansson-2.11.ebuild]: Please make sure to `git pull` and update your ebuilds to latest changes from Gentoo. In this case, you've lost `~amd64-fbsd` keywords.

**Comment by mgorny** [comment]
Ping.

**Comment by Zer0-One** [comment]
@mgorny all good?

**Comment by mgorny** [comment]
Thanks a lot. Tests pass, I'm going to merge it now.

For a future change, please consider removing .la files.

**Pull request closed without merging**